CMakeLists.txt

```diff
@@ -74,9 +82,9 @@ elseif(CMAKE_Fortran_COMPILER_ID STREQUAL "Intel")
   set(r8_flags "${r8_flags} -no-prec-div -no-prec-sqrt")
 
   # Copied from MOM5/bin/mkmf.template.nci
-  set(CMAKE_Fortran_FLAGS "${CMAKE_Fortran_FLAGS} -fno-alias -safe-cray-ptr -fpe0 -ftz -assume byterecl -i4 -r8 -traceback -nowarn -check noarg_temp_created -assume nobuffered_io -convert big_endian -grecord-gcc-switches -align all")
-  set(CMAKE_Fortran_FLAGS_RELEASE "-g3 -O2 -xCORE-AVX2 -debug all -check none")
-  set(CMAKE_Fortran_FLAGS_DEBUG "-g3 -O0 -debug all -check -check noarg_temp_created -check nopointer -warn -warn noerrors -ftrapuv")
+  set(CMAKE_Fortran_FLAGS "${CMAKE_Fortran_FLAGS} -fno-alias -safe-cray-ptr -fpe0 -ftz -assume byterecl -i4 -r8 -nowarn -check noarg_temp_created -assume nobuffered_io -convert big_endian -grecord-gcc-switches -fp-model precise -fp-model source -align all")
+  set(CMAKE_Fortran_FLAGS_RELEASE "-g3 -O2 -xCORE-AVX2 -debug all -check none -traceback")
+  set(CMAKE_Fortran_FLAGS_DEBUG "-g3 -O0 -debug all -check -check noarg_temp_created -check nopointer -warn -warn noerrors -ftrapuv -traceback")
```

Comment:
Does removing `-traceback` from the common compiler flags moving it to the config specific compiler flags have a practical benefit?

Reply:
It just means the flags now match the mom5 cmake exactly. Happy to revert that if you prefer.

Reply:
Yes, please move them back.

Reply:
Done in [c55a9c3](https://github.com/ACCESS-NRI/FMS/pull/12/commits/c55a9c3d535d4fb6ffeebc928f3317345cc979f5)